tests/unittests/Foundation/NSFileManagerTests.mm

```diff
@@ -14,18 +14,52 @@
 //
 //******************************************************************************
 
-#include <sys/types.h>
-#include <sys/stat.h>
-#include <TestFramework.h>
-#include <time.h>
-#include <stdio.h>
-#include <errno.h>
-#include <Foundation/NSFileManager.h>
-#include <Foundation/NSNumber.h>
-#include <Foundation/NSDate.h>
-#include <Foundation/NSURL.h>
-#include "stdlib.h"
-#include "windows.h"
+#import <sys/types.h>
+#import <sys/stat.h>
+#import <TestFramework.h>
+#import <time.h>
+#import <stdio.h>
+#import <errno.h>
+#import <Foundation/NSFileManager.h>
+#import <Foundation/NSNumber.h>
+#import <Foundation/NSDate.h>
+#import <Foundation/NSURL.h>
+#import <Foundation/NSFileHandle.h>
+#import <stdlib.h>
+#import <windows.h>
+
+static NSString* getPathToFile(char const* filePath) {
+    char fullPath[_MAX_PATH];
+    int len = GetModuleFileNameA(NULL, fullPath, _MAX_PATH);
+    char* ptrToLastEntry = strrchr(fullPath, '\\');
```

Comment:
pre-cache this, and simply use `[_cachedModulePath stringByAppendingPathComponent:filePathWhichIsNowAnNSString]`